test/components/partner_preview_component_test.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class PartnerPreviewComponentTest < ViewComponent::TestCase
+  def test_component_renders_something_useful
```

Comment:
actually i think i can meaningfully throw a couple tests in this one just not the other one

Reply:
quinn came out to bat for selenium actually and thinks our test config needs some love and that capybara is acutally great.

if we need to switch driver theres this too: https://github.com/rubycdp/cuprite but they warned it might not be worth the time investment to switch

Reply:
no tests was a copout, have added something basic